src/main/java/net/sf/oval/internal/util/ReflectionUtils.java

```diff
@@ -211,6 +212,31 @@ public static Method getGetterRecursive(final Class<?> clazz, final String prope
       return getGetterRecursive(superclazz, propertyName);
    }
 
+   public static List<Method> getInterfaceMethods(final Method method) {
+      // static methods cannot be overridden
+      if (isStatic(method)) {
+         return Collections.emptyList();
+      }
+
+      final Set<Class<?>> interfaces = getInterfacesRecursive(method.getDeclaringClass(), Collections.emptySet(), Collections.emptySet());
```

Comment:
Please extend the method with two parameters `final Set<Class<?>> includedInterfaces, final Set<Class<?>> excludedInterfaces` and pass them to `getInterfacesRecursive()` instead of the empty Sets. In the AnnotationsConfigurer then just pass  additionally pass `classCfg.includedInterfaces, classCfg.excludedInterfaces` when invoking this method.